Sources/ObjCSupport/LOKInsetLayout.swift

```diff
@@ -11,24 +11,28 @@ import CoreGraphics
 @objc open class LOKInsetLayout: LOKBaseLayout {
     @objc public let insets: EdgeInsets
     @objc public let alignment: LOKAlignment
+    @objc public let flexibility: LOKFlexibility?
     @objc public let viewClass: View.Type
     @objc public let sublayout: LOKLayout
     @objc public let configure: ((View) -> Void)?
 
     @objc public init(insets: EdgeInsets,
                       alignment: LOKAlignment? = nil,
+                      flexibility: LOKFlexibility? =nil,
                       viewReuseId: String? = nil,
                       viewClass: View.Type? = nil,
                       sublayout: LOKLayout,
                       configure: ((View) -> Void)? = nil) {
         self.insets = insets
         self.sublayout = sublayout
         self.alignment = alignment ?? .fill
+        self.flexibility = flexibility
```

Comment:
We should probably make this non-optional and have it reflect the actual flexibility value that would be used.

Reply:
Made `flexibility` here non-optional. But the parameter is still optional (in a different sense of the word) in InsetLayout's constructor.